Source/Fuse.Common/Tests/FuseTest/TestFramebuffer.uno

```diff
@@ -0,0 +1,67 @@
+using Uno;
+using Uno.Compiler;
+using Uno.Testing;
+
+using OpenGL;
+
+using Fuse;
+
+namespace FuseTest
+{
+	public class TestFramebuffer : IDisposable
+	{
+		internal TestFramebuffer(int2 size)
+		{
+			Framebuffer = FramebufferPool.Lock(size, Uno.Graphics.Format.RGBA8888, true);
+		}
+
+		public void Dispose()
+		{
+			FramebufferPool.Release(Framebuffer);
+		}
+
+		/**
+			Reads a pixel form the last CaptureDraw.
+		*/
+		float4 ReadDrawPixel(int2 pos)
+		{
+			var prevFramebuffer = GL.GetFramebufferBinding();
+			GL.BindFramebuffer(GLFramebufferTarget.Framebuffer, Framebuffer.RenderTarget.GLFramebufferHandle);
+
+			try
+			{
+				var temp = new byte[4];
+				GL.ReadPixels(pos.X, Framebuffer.Size.Y - 1 - pos.Y, 1, 1, GLPixelFormat.Rgba, GLPixelType.UnsignedByte, temp);
+				return float4(temp[0] / 255.0f,
+					temp[1] / 255.0f,
+					temp[2] / 255.0f,
+					temp[3] / 255.0f);
+			}
+			finally
+			{
+				GL.BindFramebuffer(GLFramebufferTarget.Framebuffer, prevFramebuffer);
+			}
+		}
+
+		public void AssertPixel(int2 pos, float4 expectedColor, float tolerace = float.ZeroTolerance, [CallerFilePath] string filePath = "", [CallerLineNumber] int lineNumber = 0, [CallerMemberName] string memberName = "")
```

Comment:
Would it make sense to put the `expectedColor` first as that is how the other `Assert.*` methods are ordered?